### DISCUSSION TITLE
下载文件夹时报这个错是啥原因

### DISCUSSION THREAD

**Discussion by nitsort** [AlistGo/alist#1118] [General]
右键文件夹-下载，然后一直卡在正在下载

![image](https://user-images.githubusercontent.com/23069540/168863101-ca0794f7-b5f5-4fee-9be5-d975b9900ecf.png)

控制台有如下报错

![image](https://user-images.githubusercontent.com/23069540/168863752-0631fb04-f001-4c31-8460-ce561e46df91.png)

**Comment by anwen-anyi** [comment]
这是打包下载了吧?推荐使用推送到Ar2下载

进你想下载的文件夹里面进去  右上角设置成列表模式 然后右键多选  选择你需要下载的  然后右键推送到Ar2

当然了前提是你先配置好Ar2再下载配置也很简单吧

**Comment by nitsort** [comment]
翻到解决办法了👀https://github.com/Xhofe/alist/discussions/186#discussioncomment-1741818
